devLib/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# CMakeLists.txt for devLib
+
+cmake_minimum_required(VERSION 3.10)
+project(devLib)
+
+# Source files
+set(SRC
+    ds1302.c maxdetect.c piNes.c
+    gertboard.c piFace.c
+    lcd128x64.c lcd.c
+    scrollPhat.c
+    piGlow.c
+)
+
+# Headers
+set(HEADERS
+    ds1302.h maxdetect.h piNes.h
+    gertboard.h piFace.h
+    lcd128x64.h lcd.h
+    scrollPhatFont.h scrollPhat.h
+    piGlow.h
+)
+
+# Compiler flags
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wformat=2 -Winline -pipe -fPIC")
```

Comment:
I think `-fPIC` is not really needed here as position independent code seems to be the default for shared libraries with CMake: https://cmake.org/cmake/help/v3.30/prop_tgt/POSITION_INDEPENDENT_CODE.html#prop_tgt:POSITION_INDEPENDENT_CODE.